### ISSUE TITLE
Template - Add a way to detect whether a link has a thumbnail or not

### ISSUE THREAD

**Issue by kalvn** [shaarli/Shaarli#487]
In my theme, I had the following issue: https://github.com/kalvn/Shaarli-Material/issues/31

And I cannot properly fix it as I put the thumbnail in a container but I have no clean way to know whether a link has a thumbnail or not. Therefore, I cannot apply different styling rules depending on this, and since I display the image as a left float, it cause issues if the description of the link includes a background or a border (as it's the case for markdown blockquotes).

And since the thumbnail is populated through a RainTPL filter, I have no way of testing if the filter will return something or not.

Would it be possible to add such possibility? For example by running the filter before the rendering and then add a new parameter like `$value.thumbnail`.

**Comment by kalvn** [comment]
As advised on the ticket mentioned above, the following code do the trick:

```
{$thumb=thumbnail($value.url)}
{if="$thumb==false"}yay{/if}
```

I guess the ticket can be closed.

**Comment by ArthurHoaro** [comment]
You can keep it open I think. That's something that's need to be done with thumbnail refactoring.

**Comment by ArthurHoaro** [comment]
Fixed by #687
